third-party/karpenter/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@kubernetes-models/karpenter",
+  "version": "3.2.1",
```

Version will be automatically bumped.

```suggestion
  "version": "0.0.0",
```

---

ah, okay thanks!

---

done!